src/components/DayPicker.jsx

```diff
@@ -288,22 +290,36 @@ export default class DayPicker extends React.Component {
   }
 
   renderNavigation() {
+    const {
+      navPrev,
+      navNext,
+    } = this.props;
+
     const isVertical = this.isVertical();
 
+    let navPrevIcon = navPrev;
+    let navNextIcon = navNext;
+    if (!navPrevIcon) {
+      navPrevIcon = isVertical ? <ChevronUp /> : <LeftArrow />;
```

Comment:
it seems like we'd want callers to be able to perform the same ternary, so they don't have to know about the "isVertical" implementation.

What would you think about making `navPrev` and `navNext` be functions, that get called with `isVertical`, so that `navPrev` would default to `vertical => (vertical ? <ChevronUp /> : <LeftArrow />)`?